src/backends/raylib_backend.zig

```diff
@@ -279,21 +302,84 @@ pub fn textureCreate(_: *RaylibBackend, pixels: [*]u8, width: u32, height: u32,
 }
 
 pub fn textureCreateTarget(self: *RaylibBackend, width: u32, height: u32, interpolation: dvui.enums.TextureInterpolation) !*anyopaque {
-    _ = self;
-    _ = width;
-    _ = height;
-    _ = interpolation;
-    return error.textureError;
+    const id = c.rlLoadFramebuffer(); // Load an empty framebuffer
+    if (id == 0) {
+        return error.textureError;
+    }
+
+    c.rlEnableFramebuffer(id);
+    defer c.rlDisableFramebuffer();
+
+    // Create color texture (default to RGBA)
+    const texid = c.rlLoadTexture(null, @intCast(width), @intCast(height), c.PIXELFORMAT_UNCOMPRESSED_R8G8B8A8, 1);
+    switch (interpolation) {
+        .nearest => {
+            c.rlTextureParameters(texid, c.RL_TEXTURE_MIN_FILTER, c.RL_TEXTURE_FILTER_NEAREST);
+            c.rlTextureParameters(texid, c.RL_TEXTURE_MAG_FILTER, c.RL_TEXTURE_FILTER_NEAREST);
+        },
+        .linear => {
+            c.rlTextureParameters(texid, c.RL_TEXTURE_MIN_FILTER, c.RL_TEXTURE_FILTER_LINEAR);
+            c.rlTextureParameters(texid, c.RL_TEXTURE_MAG_FILTER, c.RL_TEXTURE_FILTER_LINEAR);
+        },
+    }
+
+    c.rlTextureParameters(texid, c.RL_TEXTURE_WRAP_S, c.RL_TEXTURE_WRAP_CLAMP);
+    c.rlTextureParameters(texid, c.RL_TEXTURE_WRAP_T, c.RL_TEXTURE_WRAP_CLAMP);
+
+    c.rlFramebufferAttach(id, texid, c.RL_ATTACHMENT_COLOR_CHANNEL0, c.RL_ATTACHMENT_TEXTURE2D, 0);
+
+    // Check if fbo is complete with attachments (valid)
+    if (!c.rlFramebufferComplete(id)) {
+        return error.textureError;
+    }
+
+    self.frame_buffers.put(texid, id) catch unreachable;
+    self.texture_sizes.put(texid, .{ .w = @floatFromInt(width), .h = @floatFromInt(height) }) catch unreachable;
+
+    self.renderTarget(@ptrFromInt(texid));
+    c.ClearBackground(c.BLANK);
+    self.renderTarget(null);
+
+    return @ptrFromInt(texid);
 }
 
 pub fn renderTarget(self: *RaylibBackend, texture: ?*anyopaque) void {
```

Comment:
it might be worthwhile leaving a comment here explaining that passing in a valid texture begins texture mode, and that passing in a null ends it. It took me a second to understand what was happening here.

Reply:
Yes it's a bit weird but that's how opengl does it, so it kind of makes sense.